spec/requests/claims_spec.rb

```diff
@@ -61,6 +61,23 @@
         expect(response).to redirect_to(root_path)
       end
     end
+
+    context "when a claim is completed" do
```

Comment:
I think it would be good to be more specific with this context/behaviour. It's more like "when the user reaches the confirmation page after completing their claim"